src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -188,28 +194,31 @@ public override void OnActionExecuting(ActionExecutingContext context)
             base.OnActionExecuting(context);
         }
 
-        // TODO: FACAVAL
-        //[Route("admin/extensions/{name}/{*extra}")]
-        //[HttpGet]
-        //[HttpPost]
-        //[AllowAnonymous]
-        //public async Task<HttpResponseMessage> ExtensionWebHookHandler(string name, CancellationToken token)
-        //{
-        //    var provider = _scriptHostManager.BindingWebHookProvider;
-
-        //    var handler = provider.GetHandlerOrNull(name);
-        //    if (handler != null)
-        //    {
-        //        string keyName = WebJobsSdkExtensionHookProvider.GetKeyName(name);
-        //        if (!this.Request.HasAuthorizationLevel(AuthorizationLevel.System, keyName))
-        //        {
-        //            return new HttpResponseMessage(HttpStatusCode.Unauthorized);
-        //        }
-
-        //        return await handler.ConvertAsync(this.Request, token);
-        //    }
-
-        //    return new HttpResponseMessage(HttpStatusCode.NotFound);
-        //}
+        [HttpGet]
+        [HttpPost]
+        [Authorize(AuthenticationSchemes = AuthLevelAuthenticationDefaults.AuthenticationScheme)]
+        [Route("webhookextensions/handler/{name}/{*extra}")]
```

Comment:
How about a route `api/extensions/{name}/{*extra}`? The main goal I think is to move this off of the admin API since it really isn't an admin operation like the others.

Reply:
I like moving it under `api`, but wanted to have something specific enough to avoid conflicting with routes customers may want to use for their functions.